### ISSUE TITLE
Profile: Signing in

### ISSUE THREAD

**Issue by mehranhydary** [unicef/bounties#15]
@rymksotnini - hi there!

I noticed that on https://yearn.finance/ that there is a sign in method that allows the user to access the platform with multiple wallets. When you have some time tomorrow - can you look into the following: 

1. Are they using a library here to import all of the wallets: https://yearn.finance/; I think I found something here: https://github.com/iearn-finance/iearn-finance/blob/3874a7b1f9b0d7abe996dbf8dfa0765d6da1795b/src/components/unlock/unlock.jsx... doesn't seem like it's a library
2. Is it possible to do something similar for the Bounties platform? I understand right now we can only use MetaMask... what effort is required to get something like the Yearn Finance wallet connect implementation

Bonus: Seems like https://uniswap.exchange/ also did something similar. See some of the code here: https://github.com/Uniswap/uniswap-interface/blob/8fd894f2d1b03921c5519c050aed164343c47fb1/src/components/WalletModal/index.tsx. You may have to trace through a bit more.

**Comment by mehranhydary** [comment]
_After some more investigation, I think we may have to create our own sign in module for the dapps we work with... set up some time with me this week and we can discuss further!_

**Comment by rymksotnini** [comment]
@mehranhydary  - Hi!
After a deeper look at the whole iearn-finance project, I found the configuration of the different connectors under the the store folder [https://github.com/iearn-finance/iearn-finance/tree/3874a7b1f9b0d7abe996dbf8dfa0765d6da1795b/src/stores](url). So it wasn't as automated as I thought it was, using the web3-react library. I also found another implementation [https://codesandbox.io/s/p5b66?file=/pages/index.tsx:7055-7064](url). This simple project focuses mainly on the different wallet sign in that we want to achieve and helped me understand the steps to follow in configuring the connectors.
So, as you said, we actually need to develop our own module.
